cmd/influxd/server_integration_test.go

```diff
@@ -1188,7 +1193,6 @@ func TestSingleServer(t *testing.T) {
 	nodes := createCombinedNodeCluster(t, testName, dir, 1, 8090, nil)
 
 	runTestsData(t, testName, nodes, "mydb", "myrp")
-	runTest_rawDataReturnsInOrder(t, testName, nodes, "mydb", "myrp")
```

Comment:
Obviously this will not be part of the final commit, just to speed up my testing.